cloud_formation/configs/idindexing.py

```diff
@@ -153,32 +166,17 @@ def pre_init(bosslet_config):
 
 def post_init(bosslet_config):
     """Create step functions."""
-    names = bosslet_config.names
+    role = 'StatesExecutionRole-us-east-1 '
```

Comment:
Will the role eventually be configurable?

Reply:
Configurable? It already is...
When Step Functions were first introduced they had not update IAM to allow use to create roles for Step Functions, and so we've continued to use the default role created by AWS even though we can now create custom IAM Step Function roles.